src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "../reseat/reseat.scss";
+@import "./untiles/mixins";
```

Comment:
Typo in the import path for 'mixins'. It should be './utilities/mixins'.

Reply:
![image](https://user-images.githubusercontent.com/87493308/236918160-85bb3325-43be-4041-8446-9ec446fbab89.png)
I need exactly a two-bracket wrapper, I changed the name of the folder as a whole, in the future I will use a single-bracket wrapper, I'm doing the standard of the task))